certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -2286,6 +2497,56 @@
         "deprecated": false,
         "x-ms-visibility": "internal"
       }
+    },
+    "/build_number": {
+      "get": {
+        "tags": [
+          "StaticResponse"
+        ],
+        "description": "Identifier for the currently deployed build.",
+        "summary": "Show build Number (For reference only. Not meant for including in a flow for execution)",
```

Comment:
@sriharip-docusign The 'summary' value goes over the character limit '80'. Please fix. thank you